sycl/test-e2e/InOrderEventsExt/set_external_event.cpp

```diff
@@ -8,16 +8,13 @@
 
 constexpr size_t N = 1024;
 
-int main() {
-  sycl::context Ctx;
-  sycl::device Dev = Ctx.get_devices()[0];
-
-  sycl::queue Q1{Ctx, Dev, {sycl::property::queue::in_order{}}};
-  sycl::queue Q2{Ctx, Dev, {sycl::property::queue::in_order{}}};
+int check_work(sycl::queue &Q1, sycl::queue &Q2) {
+  std::cout << "Checking ext_oneapi_set_external_event for a workload."
+            << std::endl;
 
   sycl::buffer<int> DevDataBuf{sycl::range{N}};
   sycl::accessor DevData{DevDataBuf};
-  int *HostData = (int *)malloc(N * sizeof(int) * 10);
+  int *HostData = new int[N * sizeof(int) * 10];
```

Comment:
Isn't `sizeof(int)` implied by `new int`?

Reply:
Right you are! Fixed.